be/src/common/daemon.cpp

```diff
@@ -289,6 +289,11 @@ void Daemon::start() {
     CHECK(st.ok()) << st.to_string();
 
     if (config::enable_metric_calculator) {
+        CHECK(DorisMetrics::instance()->is_inited())
```

Comment:
Can we just skip calculate **system metrics** in calculate_metrics_thread when `enable_system_metrics ` is false?

Reply:
If some check failed could also cause system metrics init failed,such as disk or inet interface.
So using a init field seems better.